src/Domain/Creators/TimeDependence/UniformRotationAboutZAxis.cpp

```diff
@@ -69,9 +68,9 @@ UniformRotationAboutZAxis<MeshDim>::functions_of_time() const noexcept {
   // We use a `PiecewisePolynomial` with 2 derivs since some transformations
   // between different frames for moving meshes can require Hessians.
   result[function_of_time_name_] =
-      std::make_unique<FunctionsOfTime::PiecewisePolynomial<2>>(
-          initial_time_,
-          std::array<DataVector, 3>{{{0.0}, {angular_velocity_}, {0.0}}});
+      std::make_unique<FunctionsOfTime::PiecewisePolynomial<3>>(
```

Comment:
I asked @nilsdeppe about the comment above, since I'm actually confused by it. I only made this change because the tests were failing, and this fixed it.

Reply:
After discussing with @nilsdeppe, I updated the comment to be correct. To clarify, the issue is that the reader assumes functions of time are third order piecewise polynomials, so to use the new reader to read in a rotation time dependence, as I will need to do for BBH evolutions, the rotation time dependence must also be a third order piecewise polynomial.